components/core/src/package/install.rs

```diff
@@ -1280,17 +1280,28 @@ mod test {
         expected.insert("FOO".to_string(), "bar".to_string());
         expected.insert("JAVA_HOME".to_string(), "/my/java/home".to_string());
         expected.insert(
-                        "PSModulePath".to_string(),
-                        env::join_paths(vec![
-            fs::fs_rooted_path(&pkg_prefix_for(&pkg_install).join("/should/not/be/ignored"), &fs_root_path),
-            PathBuf::from("c:/my/dir"),
-            fs::fs_rooted_path(
-                &pkg_prefix_for(&pkg_install).join("/should/really/not/be/ignored"),
-                &fs_root_path,
-            ),
-        ]).unwrap()
-                        .to_string_lossy()
-                        .into_owned(),
+            "PSModulePath".to_string(),
+            env::join_paths(vec![
+                fs::fs_rooted_path(
```

Comment:
I was looking at this change and also clippy's complaint with the old code and the test failure you are seeing. I think clippy's suggestion about `Path::new` is the way to go:

```
        expected.insert(
                        "PSModulePath".to_string(),
                        env::join_paths(vec![
            fs::fs_rooted_path(&PathBuf::from("/should/not/be/ignored"), &fs_root_path),
            PathBuf::from("c:/my/dir"),
            fs::fs_rooted_path(
                &PathBuf::from("/should/really/not/be/ignored"),
                &fs_root_path,
            ),
        ]).unwrap()
                        .to_string_lossy()
                        .into_owned(),
        );
```

Reply:
okay sure ill make the changes